CONTRIBUTING.md

```diff
@@ -1,79 +1,114 @@
 # Contributing to CodeBooker development
 
-We welcome Pull Requests (PR's) as well as having people working on fixing it's current issues. If you are coming to this project new.
-If you notice an issue with this app, please feel free to open an issue.
+This project welcomes contributors to help improve its codebase by identifying shortcomings in the project, highlighting bugs, and issues in the documentation.
 
-A lot of developers prefer asking if they could contribute or be assigned a task as a reply to a reported issue in the Issues tab. This is great.
+## What to contribute
 
-Typically, if you think it will take you longer than 72 hours after having been assigned to the issue you will work on to submit a PR, please
-let us know that way issues do not go stale.
+Anyone with basic programming skills down to grammer skills can contribute immensely to this project.
+Areas requiring contribution includes:
 
-Some tasks may not be beginner friendly even if tagged `good first issue`.. so try and judge the task accordingly.
-If you are a beginner, there are smaller tasks a beginner can work on such as style issues.
+- **Error checks**: Identify any mispellings or errors in documentation? 
 
-You should already be familiar with React 17 and or 18, JavaScript ES5/ES6/ES7.
+- **Documentation**: Possess strong documentation skills, and believe you can help elevate this projects' documentation even better? 
 
-## Style
+- **Code contribution**: Identify a bug in this app or thought of a new feature that could plunge this app to a new dimension? 
+
+## How to contribute
+
+Developers often derive more convenience in seeking permission from the project to contribute, so these next headings will get you started in no time contributing.
+
+## Select an issue
+
+If you are not sure what to work on, review the issues list first. There are also TODO's listed in the included [TODO.md](./TODO.md) file. 
+
+@gbowne1 and other project maintainers will assign users to issues on a first come, first serve basis. If you would like to work on an issue, feel free to indicate by tagging the mentioned persons.
+
+## Create an issue
+
+If you notice a typo, slight bug in the code and there are no lodged complaints on this yet, select the issues icon at the top, and on the right side of the next page select new issues and lodge your complain.
```

Comment:
This could be easier to write if we change `If you notice a typo, slight bug in the code and there are no lodged complaints on this yet, select the issues icon at the top, and on the right side of the next page select new issues and lodge your complain.` 

to:

`Have you noticed something that did not look or work right in our applicaton, or a typographical error, wrong syntax, grammar, spelling or other issue?  

- Check to see if an issue has been created for whatever you have found  in <https://github.com/gbowne1/codebooker/issues>
- If there is not a issue that has already been created, Click on the green [New Issue] button.
- Describe your issue as well as you can:
  - include code snippits where the issue
  - include screen shots
  - include a short video
  
  If the issue appears to be a browser, operating system, device specific issue, let us know what those are.

Reply:
Okay this is a much better alternative, will get this fixed.

Reply:
@Chizobaonorh Ok. Let me know when you have updates.